format.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+$DIR/node_modules/.bin/tslint  -c tslint.json 'src/**/*.ts'
```

Comment:
# Requested Change

While this is good for people using something with bash, it would be good if people on Windows could run this script as well. You can actually add scripts in the `package.json` files (which is what I was referring to as `npm` scripts). 

You can see in the `scripts` section one of the ones Bea already wrote for this project:

`"watch": "npm run build -- -w"`

So running `npm run watch` would invoke this script. Something I forgot to mention before, we should also be linting the tests. And `fix` scripts would be great too. You can see how to append options to an `npm` script, and how to run them sequentially, [here](https://github.com/Right2Drive/ease/blob/master/package.json). There are scripts to run linting against test and src independently, and one to run both, and `lint:fix` scripts as well.